constants/Backend.js

```diff
@@ -18,7 +18,6 @@ export const storage = {
 //export const apiRoot = 'http://10.0.4.9:8080'
 //export const apiRoot = 'http://192.168.1.102:8080'
 //export const apiRoot = 'http://192.168.2.244:8080'
```

Comment:
Let's remove them

Reply:
I try to import timezone in custom.js, but I found it's must be default value because ./action not below App.js.
Or, think can change thought. I will try to place timezone in this.context.